.github/workflows/configs-go/config-release.yml

```diff
@@ -12,3 +12,7 @@ goos: linux
 goarch: amd64
 dir: internal/builders/go/
 binary: slsa-builder-go-{{ .Os }}-{{ .Arch }}
+# ldflags: -s -w is used for reducing the size of the binary.
```

Comment:
Can you update the comment to be more specific, e.g. `-s -w ldflags remove the debugging information from the binary, which reduce the size of the binary.`?

Reply:
I did that.  Thanks!